gsuite/resource_user.go

```diff
@@ -18,12 +19,25 @@ func flattenUserName(name *directory.UserName) map[string]interface{} {
 	}
 }
 
+func normalizeJSON(jsonString interface{}) string {
+	if jsonString == nil || jsonString == "" {
+		return ""
+	}
+	var j interface{}
+	err := json.Unmarshal([]byte(jsonString.(string)), &j)
+	if err != nil {
+		return fmt.Sprintf("Error parsing JSON: %s", err)
```

Comment:
This seems a bit dangerous. If we return a string with an error in it, we confuse people more and potentially store the error message. I think we want to return the usual tuple with err and value, and then have the flatten function do the same, and bubble it up all the way into the calling code so it can do the usual provider err dance.